projects/sandbox/src/app/problem-viewer/problem-viewer.component.html

```diff
@@ -236,17 +263,19 @@
         <ng-template ngPluralCase="=1"> record</ng-template>
         <ng-template ngPluralCase="other"> records</ng-template>
       </span>
-      <ng-content
-        *ngTemplateOutlet="
-          tmpRecordAnalysisList;
-          context: {
-            analyses: problemPattern.recordAnalysisList,
-            problemTitle: problemPattern.problemPatternDescription.problemPatternTitle,
-            datasetId: datasetId,
-            ppId: problemPattern.problemPatternDescription.problemPatternId
-          }
-        "
-      ></ng-content>
+      <ng-container *ngIf="!problemPattern.closed">
```

Comment:
Is there a boolean method `open`? It would be easier to read if it were `ngIf="problemPattern.open"`

Reply:
There is no boolean method - just the "closed" field in the augmented model.

Reply:
> Is there a boolean method `open`? It would be easier to read if it were `ngIf="problemPattern.open"`

There is no boolean method - just the "closed" field in the (augmented) model - and that's been there since problem patterns were first introduced!